hnn_core/gui/gui.py

```diff
@@ -590,18 +661,13 @@ def _run_button_clicked(b):
                 self._log_out, self.drive_widgets, self.simulation_data,
                 self.widget_dt, self.widget_tstop, self.widget_ntrials,
                 self.widget_backend_selection, self.widget_mpi_cmd,
-                self.widget_n_jobs, self.params, self.plot_outputs_list,
-                self.plot_dropdowns_list, self._simulation_status_bar,
+                self.widget_n_jobs, self.params, self._simulation_status_bar,
                 self._simulation_status_contents,
-                self.connectivity_widgets, b)
+                self.connectivity_widgets, self.analysis_config, b)
 
-        def _handle_viz_layout_change(layout_option):
-            return _initialize_viz_window(
-                self._viz_grid,
-                self._visualization_window, self.simulation_data,
-                self.plot_outputs_list, self.plot_dropdowns_list,
-                self.layout['visualization_window'],
-                layout_option=layout_option.new)
+        def _handle_viz_layout_change(_):
```

Comment:
what does the underscore do when passed to a function? I have never seen this. Is it a new Python feature or existed before?

Reply:
It does nothing basially, more like a placeholder. I use it as `_` is not treated as a sort of "variable" that should be accessed somewhere in vscode so the color won't change.  https://stackoverflow.com/questions/5893163/what-is-the-purpose-of-the-single-underscore-variable-in-python